Tests/configure_and_test_integration_instances.py

```diff
@@ -1332,6 +1389,7 @@ def main():
     build = Build(options_handler())
 
     configure_servers_and_restart(build)
+    disable_instances(build)
```

Comment:
Can you please explain why this line was moved?

Reply:
I want to verify that there are no configured. integrations before I start the test-module.

Reply:
But do we want it also after all of the instance testing is finished?

Reply:
No actual need, no instances are created in this process it only runs test-module on them